dirsrvtests/tests/suites/password/pwdPolicy_warning_test.py

```diff
@@ -140,7 +140,7 @@ def local_policy(topology_st, add_user):
 
     log.info("Setting fine grained policy for user ({})".format(USER_DN))
 
-    subprocess.call(['%s/dsconf' % topology_st.standalone.get_sbin_dir(),
+    subprocess.call(['%s/dsconf' % topology_st.standalone.get_bin_dir(),
```

Comment:
But dsconf is in /usr/sbin by default.  Maybe this needs to handle both locations?

Reply:
find $DSDIR | grep dsconf
/home/progier/sb/i4648/tst/ci-install/bin/dsconf
 I will double check why it it there in my build ...

Reply:
Yeah, in rpm builds it goes in `/usr/sbin/`: https://github.com/389ds/389-ds-base/blob/master/rpm/389-ds-base.spec.in#L654

Reply:
Ok it is a problem in my environment - I must revert that change